packages/keystone/fields/AddressPartWithType/views/Cell.js

```diff
@@ -0,0 +1,13 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react'
+
+export default function AddressPartWithTypeCell ({ data }) {
+    const { name = '', typeShort = '', typeFull = '' } = JSON.parse(data)
+
+    return (
+        <div>
+            <div>{typeFull} · {typeShort}</div>
```

Comment:
Do we need this extra div if both parts are empty?

Reply:
Yes, why not
<img width="1621" alt="image" src="https://user-images.githubusercontent.com/25384290/203924110-c798a56d-992f-4b44-b469-31e937c6e919.png">